docs/app/documentation/component-docs/input/input-docs.component.html

```diff
@@ -1,28 +1,48 @@
-<h2>Input</h2>
+<h2 id="input" class="docs-header-link"> 
+    <a class="docs-markdown-a" aria-describedby="input" href="/input#input">
+        <span class="sap-icon--chain-link"></span>
+    </a>
+        Input      
+</h2>
 <component-example [name]="'ex1'">
     <fd-input-example></fd-input-example>
 </component-example>
 <code-example [exampleFiles]="inputsFormHtml"></code-example>
 
 <separator></separator>
 
-<h2>Input Help elements</h2>
+<h2 id="input-help-elements" class="docs-header-link">
+    <a class="docs-markdown-a"aria-describedby="input-help-elements" href="/input#input-help-elements">
+        <span class="sap-icon--chain-link"></span>
+    </a>
+        Input Help elements
+</h2>
```

Comment:
I think it will be better to create component which will be named `docs-section-title`. You would be able to pass `[siteUrl]` and `[sectionUrl]`. There can be also `<ng-content>` inside template for title section name. It will save a lot of space and time.